### ISSUE TITLE
New materials

### ISSUE THREAD

**Issue by bartoszkubicki** [mage2tv/issues#50]
Hello @Vinai! Your videos are the best materials, which I have found, espacially about FPC and generally javascript. It is good, that you show use-cases for different featues and also don't dive deeply into implemenation details, as they are not necessary for day-to-day development.

Are you planning to add more materials? There as a few empty categories to fill :)

**Comment by Vinai** [comment]
Hi @bartoszkubicki, thank you for the issue!
Yes, I am working on more content.

I apologize for the long pause. I intend to get content out quicker, but well, as the saying goes, the road to hell is paved with good intentions...
Other work left me more exhausted than I had planned, so I couldn't keep up the relatively high volume of output I started with. 
Also, I needed to implement coupons in Mage2 TV for a bulk deal I have with a company. That also took some time I originally had planned on for new content production. But that is done now,

I'm now striving for a more balanced, sustainable approach.
The next "chapter" (that's what I call it internally in my mind) GraphQL is exciting and fun, and I'm really looking forward to sharing it with you.

Thank you for your loyalty and your patience!

Vinai
